### PR TITLE
Add missing typing_extensions to setup.cfg

### PR DESCRIPTION
**Pull request by ravwojdyla**
Example of failure (from sgkit_vcf): https://github.com/pystatgen/sgkit-vcf/pull/11/checks?check_run_id=1112654920, reason `typing_extensions` are not in the ~requirements.~ setup.cfg.

Re: #255

### CODE REVIEW

**Comment by ravwojdyla** [comment]
Looks like py3.7 doesn't have Literal, I might need to add `typing_extensions` to requirements in the end.

File: setup.cfg [hunk 1]
```diff
@@ -31,6 +31,7 @@ install_requires =
     scipy
     zarr
     numba
+    typing-extensions
```
Comment (eric-czech) [inline setup.cfg:34]: This doesn't need to go into requirements.txt or the mypy requirements?

**Review by eric-czech — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (ravwojdyla) [reply setup.cfg:34]: @eric-czech it's already in the `requirements.txt` (thus this bug), but good point about `mypy requirements`, missed that one. We should really automated this via https://github.com/pystatgen/sgkit/issues/39

Comment (ravwojdyla) [reply setup.cfg:34]: Fixed

**Comment by jeromekelleher** [comment]
Seems uncontroversial so merging.

**Pull request merged**